src/content/docs/release-notes/mobile-release-notes/net-maui-release-notes/net-maui-agent-116.mdx

```diff
@@ -0,0 +1,11 @@
+---
+subject: .NET MAUI agent
+releaseDate: '2024-12-10'
+version: 1.1.4
```

Comment:
The version is not matching with the file name. Is 1.1.6 is the correct version?

Reply:
@sujitnewrelic can you update it?